src/SourceBuild/content/global.json

```diff
@@ -5,6 +5,6 @@
   "msbuild-sdks": {
     "Microsoft.Build.NoTargets": "3.7.0",
     "Microsoft.Build.Traversal": "3.4.0",
-    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24075.2"
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24102.4"
```

Comment:
We shouldn't just update this without updating the rest of the bootstrap artifacts.  You risk breaking the CI legs that build with the Previous SDK

Reply:
I think @mthalman's change did update the bootstrap artifacts. https://github.com/dotnet/installer/commit/530bfbd9d4b1d2232df9c1c6e45031256602f613#diff-03917dc2c8b218f596449dea0b068769ce2aa2493bf945873e6673eddbc6e6d9R25-R26

I'll verify that the versions align, but I think the main issue was just a missed update of global.json for the arcade SDKin that PR.

Reply:
Oh right. The Arcade SDK entry wasn't updated.